heartbeat/eventext/eventext.go

```diff
@@ -30,3 +30,22 @@ func MergeEventFields(e *beat.Event, merge common.MapStr) {
 		e.Fields = merge
 	}
 }
+
+// EventCalledMetaKey is the full path to the key marking an event as cancelled.
+const EventCancelledMetaKey = "meta.__hb_evt_cancel__"
```

Comment:
Nit: Leave out the `meta.` as otherwise if you use `Put` with a `.` in the key it will create a tree and then the Get has to traverse the tree. Not having this would make the checks more efficient assuming this will happen on a lot of times.

Reply:
Yeah, it is unfortunate that this is more expensive, but otherwise it winds up in `event.Fields` which is even hackier. I'm not really worried about the perf here since that should be very fast, even though it isn't ideal. WDYT?